deal.II-toolchain/packages/petsc.package

```diff
@@ -80,10 +80,27 @@ if [ ! -z "${PARMETIS_DIR}" ]; then
         ${CONFOPTS} \
         --with-parmetis-dir=${PARMETIS_DIR} \
 	--with-metis-dir=${PARMETIS_DIR}"
+fi
+
+if [ ! -z "${SCALAPACK_DIR}" ]; then
+    cecho ${INFO} "PETSc: configuration with SCALAPACK"
+    CONFOPTS="\
+        ${CONFOPTS} \
+        --with-scalapack-dir=${SCALAPACK_DIR}"
+else
+  if [ ! -z "${PARMETIS_DIR}" ]; then
+    CONFOPTS="${CONFOPTS} --download-scalapack=1"
+  fi
```

Comment:
I added this check for ParMETIS only because in the PETSc package it checks if ParMETIS is present before installing MUMPS and ScaLAPACK.
See: [petsc.package#L76-L87](https://github.com/dealii/candi/blob/af9fff2d175f151d4dadcd4d9f39f9507c00a0fc/deal.II-toolchain/packages/petsc.package#L76-L87)

I have not tested whether this check is necessary, I just assumed that it is necessary because it was already there.

Reply:
Ah okay I see, in the original there are the downloads for mumps and scalapack only if parmetis is present without explaining it.

Reply:
Anyhow, this looks strange and should be tested if this makes sense.

Reply:
It seems like ParMATIS is not necessary for MUMPS and ScaLAPACK, therefore I removed the check.